src/table/column.rs

```diff
@@ -132,6 +133,10 @@ pub enum PgInterval {
     MinuteToSecond,
 }
 
+/// An instance of the PostgreSQL LTree value
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct PgLTree(pub(crate) String);
```

Comment:
@EstebanBorai Why do you need this type?

Reply:
Hi @ikrivosheev! This type is intended to represent the `ltree`'s extension data type!
https://www.postgresql.org/docs/current/ltree.html#LTREE-OP-TABLE

Reply:
I see, but I didn't see the use of the type)

Reply:
Im not 100% sure on the different engine types. But my goal was to represent this column type and provide implementations for each related operation on this type instance.

Reply:
For example:

```rust
// All MySQL year type field length sizes
#[derive(Debug, Clone, Eq, PartialEq)]
pub enum MySqlYear {
    Two,
    Four,
}
```

uses in `enum ColumnType`. I think you can remove the type)

Reply:
@ikrivosheev type removed!